deps/pytest-and-dbus-testing-dependencies.sh

```diff
@@ -45,7 +45,7 @@ function install {
 }
 
 # Install both python2 and python3 versions
-install python3-dbus python3-pip
+install python3-dbus python3-pip python3-lxc
 install python-gobject python-dbus python-pip
 
 pip install pydbus
```

Comment:
do we use this dep for anything nowadays?

Reply:
yes it is used by component and service tests in softwarecontainer

Reply:
would be nice to switch them to python3, but that's another story